app/src/main/java/org/mozilla/fenix/home/recentsyncedtabs/view/RecentSyncedTab.kt

```diff
@@ -0,0 +1,207 @@
+                    modifier = Modifier.fillMaxHeight()
+                ) {
+                    if (tab == null) {
+                        RecentTabTitlePlaceholder()
+                    } else {
+                        PrimaryText(
+                            text = tab.title,
+                            fontSize = 14.sp,
+                            overflow = TextOverflow.Ellipsis,
+                            maxLines = 2,
+                        )
+                    }
+
+                    Row(verticalAlignment = Alignment.CenterVertically) {
+                        if (tab == null) {
+                            Box(
+                                modifier = Modifier
+                                    .background(FirefoxTheme.colors.layer3)
+                                    .size(18.dp)
+                            )
+                        } else {
+                            Image(
+                                painter = painterResource(R.drawable.ic_synced_tabs),
+                                contentDescription = stringResource(
+                                    R.string.recent_tabs_synced_device_icon_content_description
+                                ),
+                                modifier = Modifier.size(18.dp)
+                            )
+                        }
+
+                        Spacer(modifier = Modifier.width(8.dp))
+
+                        if (tab == null) {
+                            TextLinePlaceHolder()
+                        } else {
+                            SecondaryText(
+                                text = tab.deviceDisplayName,
+                                fontSize = 12.sp,
+                                overflow = TextOverflow.Ellipsis,
+                                maxLines = 1,
+                            )
+                        }
+                    }
+                }
+            }
+
+            Spacer(modifier = Modifier.height(32.dp))
+
+            Button(
+                onClick = onSeeAllSyncedTabsButtonClick,
+                colors = ButtonDefaults.outlinedButtonColors(
+                    backgroundColor = if (tab == null) {
+                        FirefoxTheme.colors.layer3
+                    } else {
+                        FirefoxTheme.colors.actionSecondary
+                    }
+                ),
+                elevation = ButtonDefaults.elevation(
+                    defaultElevation = 0.dp,
+                    pressedElevation = 0.dp
+                ),
+                modifier = Modifier
+                    .height(36.dp)
+                    .fillMaxWidth()
+            ) {
+                if (tab != null) {
+                    Text(
+                        text = stringResource(R.string.recent_tabs_see_all_synced_tabs_button_text),
+                        textAlign = TextAlign.Center,
+                        color = FirefoxTheme.colors.textActionSecondary
```

Comment:
I have a fixed for the color problems you were seeing in https://github.com/mozilla-mobile/fenix/issues/24653.

Reply:
TY!